README.rst

```diff
@@ -255,6 +255,20 @@ Ubuntu and derivatives
 - Trisquel GNU/Linux 6 (based on Ubuntu 12.04)
 - Ubuntu 12.04/14.04/16.04
 
+Ubuntu Best Effort Support: Non-LTS releases 
+
+This script provides best-effort support for current, non-LTS 
+Ubuntu releases. If package repositories are not provided on 
+[repo.saltstack.com](http://repo.saltstack.com/#ubuntu) for 
+the non-LTS release, the bootstrap script will attempt to 
+install the packages for the most closely related LTS Ubuntu
+release instead. 
+
+For example, when installing Salt on Ubuntu 16.10, the bootstrap
+script will setup the repository for Ubuntu 16.04 from 
+[repo.saltstack.com](http://repo.saltstack.com/#ubuntu) and 
+install the 16.04 packages.
+
```

Comment:
Maybe it would be better to format links using RsT syntax? Like this:
```
text text `SaltStack corporate repository for Ubuntu`_ some other text

.. _`SaltStack corporate repository for Ubuntu`: http://repo.saltstack.com/#ubuntu
```

This will probably look more clear when the file will be rendered on GitHub.
Also, the line width of 100 chars was used before.